src/schema/v2/artworksForUser/helpers.ts

```diff
@@ -53,13 +56,11 @@ export const getNewForYouArtworkIDs = async (
     ? `maxWorksPerArtist: ${args.maxWorksPerArtist}`
     : ""
 
-  const first = args.first + args.excludeArtworkIds.length
-
   const query = {
     query: gql`
         query newForYouRecommendationsQuery {
           newForYouRecommendations(
-            first: ${first}
+            first: ${MAX_ARTWORKS}
```

Comment:
Overfetching here because we cannot pass an `offset` argument, and the page cursors from MP and Vortex are not in sync. This is something we should fix in the next iteration by moving to a REST endpoint or making it work somehow.